pages/index.vue

```diff
@@ -830,21 +809,23 @@
           });
         }
       },
-      getQueryStringFromPath() {
-        let queryString,
-            pathParsed = url.parse(this.path);
-        return queryString = pathParsed.query ? pathParsed.query : '';
+      updateUrlParams(paramString = this.queryString) {
+        if (paramString && !this.url.includes(paramString)) {
```

Comment:
Perhaps check the character that comes after the paramString, be it `&` or the end of the string - as then you know if that exact parameter string is contained in the URL - rather than relying on `includes`.